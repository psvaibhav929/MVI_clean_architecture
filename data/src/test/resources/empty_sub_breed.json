{
  "message": {
    "breed1": ["sub1", "sub2"],
    "breed2": []
  },
  "status": "success"
}