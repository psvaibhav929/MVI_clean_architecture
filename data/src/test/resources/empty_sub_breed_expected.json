{
  "dogs": [
    {
      "dogFullName": "breed1 - sub1",
      "dogBreedName": "breed1",
      "dogSubBreedName": "sub1"
    },
    {
      "dogFullName": "breed1 - sub2",
      "dogBreedName": "breed1",
      "dogSubBreedName": "sub2"
    },
    {
      "dogFullName": "breed2",
      "dogBreedName": "breed2",
      "dogSubBreedName": ""
    }
  ]
}
